tests/0150-telemetry_mock.c

```diff
@@ -39,28 +39,31 @@ typedef struct {
 } rd_kafka_telemetry_expected_request_t;
 
 static void test_telemetry_check_protocol_request_times(
-    rd_kafka_mock_request_t **requests_actual,
-    size_t actual_cnt,
+    rd_kafka_mock_cluster_t *mcluster,
     rd_kafka_telemetry_expected_request_t *requests_expected,
     size_t expected_cnt) {
         int64_t prev_timestamp = -1;
         int64_t curr_timestamp = -1;
         size_t expected_idx    = 0;
         size_t actual_idx      = 0;
-        const int buffer       = 200 /* constant buffer time. */;
+        const int buffer       = 500 /* constant buffer time. */;
```

Comment:
why did we increase the buffer time?

Reply:
test flakyness when adding consumer runs too